### ISSUE TITLE
No results whatsoever

### ISSUE THREAD

**Issue by Sequential-circuits** [thtrieu/darkflow#1022]
I trained with 49 pictures I took at high res using the command
**flow --model cfg/meu.cfg --load bin/yolo.weights --train --annotation train --dataset train --savepb --batch 1 --gpu 0.7  --load -1**

I have just one class (a deodorant), which I want the system to identify in the camera

My labels.txt contains just line line: **Old Spice**

The XML files are like this:
**<?xml version="1.0" ?>
<annotation>
<folder>images</folder>
<filename>IMG_20181101_112452.jpg</filename>
<path>/root/darkflow/images/IMG_20181101_112452.jpg</path>
<source>
    <database>Unknown</database>
</source>
<size>
    <width>3456</width>
    <height>4608</height>
    <depth>3</depth>
</size>

<segmented>0</segmented>
    <object>
    <name>Old Spice</name>
    <pose>Unspecified</pose>
    <truncated>0</truncated>
    <difficult>0</difficult>
    <polygon>
                        <x1>152</x1>
                        <y1>768</y1>
                        <x2>872</x2>
                        <y2>3992</y2>
                        <x3>2288</x3>
                        <y3>3952</y3>
                        <x4>2512</x4>
                        <y4>432</y4>
                        <x5>1512</x5>
                        <y5>216</y5>
                        <x6>904</x6>
                        <y6>328</y6>
                </polygon>
</object>
</annotation>**

My cfg file is :
**[net]
batch=1
subdivisions=1
width=416
height=416
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.001
max_batches = 10
policy=steps
steps=-1,100,20000,30000
scales=.1,10,.1,.1

[convolutional]
batch_normalize=1
filters=16
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=32
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=64
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=128
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=1

[convolutional]
batch_normalize=1
filters=1024
size=3
stride=1
pad=1
activation=leaky

###########

[convolutional]
batch_normalize=1
size=3
stride=1
pad=1
filters=1024
activation=leaky

[convolutional]
size=1
stride=1
pad=1
filters=30
activation=linear

[region]
anchors = 1.08,1.19,  3.42,4.41,  6.63,11.38,  9.42,5.11,  16.62,10.52
bias_match=1
classes=1
coords=4
num=5
softmax=1
jitter=.2
rescore=1

object_scale=5
noobject_scale=1
class_scale=1
coord_scale=1

absolute=1
thresh = .5
random=1**

After the train, where it would not below 2%, I exported the train using the command:
**flow --model cfg/meu.cfg --load -1 --savepb --gpu 1.0**

Finally, I ran the train using the command:
**flow --pbLoad built_graph/meu.pb --metaLoad built_graph/meu.meta --demo camera**

It does not recognize the deodorant: it just shows what the camera is getting and no boxes of any kind.

Any ideas?

**Comment by HSHunterR** [comment]
You can turn down the threshold(--threshold 0.1 or 0.2) to see are there have bounding boxes.
